heudiconv/convert.py

```diff
@@ -550,7 +550,7 @@ def save_converted_files(res, item_dicoms, bids_options, outtype, prefix, outnam
             # _sbref sequences reconstructing magnitude and phase generate
             # two NIfTI files IN THE SAME SERIES, so we cannot just add
             # the suffix, if we want to be bids compliant:
-            if bids_file and this_prefix_basename.endswith('_sbref'):
+            if bids_file and this_prefix_basename.endswith('_sbref') and len(suffixes)>len(echo_times):
```

Comment:
wouldn't I be more stringent while staying correct asking for `len(suffixes) == len(echo_times)*2` to match "both M and P" types closer?
Ideally there should be no guess work but first a full sweep through all the files (note that there seems `load_json` being redone more than ones, so better be RFed just to load them all ones), and see if we have both M and P pairs. If only one -- no `_rec-`, if there is some "middle ground"  (more M than P) - might be worth issuing a warning or just crash since AFAIK shouldn't happen.

Reply:
ideas @pvelasco ?

Reply:
actually I now think that `>` is enough.  I will just add a warning... I absorb this PR into #387  (will push shortly)